apps/web/app/(app)/environments/[environmentId]/integrations/actions.ts

```diff
@@ -34,7 +35,7 @@ export const deleteIntegrationAction = authenticatedActionClient
   .action(async ({ ctx, parsedInput }) => {
     await checkAuthorization({
       userId: ctx.user.id,
-      organizationId: await getOrganizationIdFromEnvironmentId(parsedInput.integrationId),
+      organizationId: await getOrganizationIdFromEnvironmentId(parsedInput.environmentId),
```

Comment:
please use `getOrganizationIdFromIntegrationId` for this, instead of adding environmentId in the args, as it adds security risks, if someone sends a personal environmentId and deletes someone else's integration.